src/js/health-records/utils/validations.js

```diff
@@ -0,0 +1,6 @@
+export function isValidDateRange(startDate, endDate) {
+  if (!startDate || !endDate) {
+    return true;
```

Comment:
Is this function supposed to return `true` if there is a null startDate or endDate?